packages/editor/src/client/blocks/richTextBlock/toolbar/linkMenu.tsx

```diff
@@ -0,0 +1,168 @@
+import React, {useState, useEffect, useContext} from 'react'
+import {useTranslation} from 'react-i18next'
+import {Button, Form, FormGroup, ControlLabel, FormControl, ButtonToolbar} from 'rsuite'
+import {Transforms, Range, Editor} from 'slate'
+import {useSlate} from 'slate-react'
+import {WepublishEditor} from '../editor/wepublishEditor'
+import {InlineFormat} from '../editor/formats'
+import {SubMenuContext} from '../../../atoms/toolbar'
+import axios from 'axios'
+
+export function LinkMenu() {
+  const editor = useSlate()
+
+  const {closeMenu} = useContext(SubMenuContext)
+
+  const [title, setTitle] = useState('')
+  const [url, setURL] = useState('')
+
+  const [selection, setSelection] = useState<Range | null>(null)
+
+  const [isValidURL, setIsValidURL] = useState(false)
+  const isDisabled = !isValidURL
+
+  const {t} = useTranslation()
+
+  useEffect(() => {
+    validateURL(url).then(value => setIsValidURL(value))
+  }, [url])
+
+  useEffect(() => {
+    setSelection(editor.selection)
+
+    const nodes = Array.from(
+      WepublishEditor.nodes(editor, {
+        at: editor.selection ?? undefined,
+        match: node => node.type === InlineFormat.Link
+      })
+    )
+    const tuple = nodes[0]
+    if (tuple) {
+      const [node] = tuple
+      setTitle((node.title as string) ?? '')
+      setURL((node.url as string) ?? '')
+    } else if (editor.selection) {
+      const text = Editor.string(editor, editor.selection)
+      setTitle(text ?? '')
+    }
+  }, [])
+
+  return (
+    <>
+      <Form fluid>
+        <FormGroup>
+          <ControlLabel>{t('blocks.richText.link')}</ControlLabel>
+          <FormControl
+            errorMessage={url && !isValidURL ? 'Invalid Link' : undefined}
```

Comment:
I suggest to work with the component https://rsuitejs.com/components/form/#Help%20Block. In my opinion it's less intrusive. Also the message should be more like "Could not validate link. Please check"